Header header
float32 heading
float32 pitch
float32 roll
float32 depth

uint8 kill
float32 horLeft
float32 horRight
float32 verLeft
float32 verRight

float32 cpuTemp
float32 cpuUsage
float32 memUsage

string coreURI